RecoEcal/EgammaClusterAlgos/interface/PFECALSuperClusterAlgo.h

```diff
@@ -144,7 +153,10 @@ class PFECALSuperClusterAlgo {
   clustering_type _clustype;
   energy_weight _eweight;
   void buildAllSuperClusters(CalibratedClusterPtrVector&, double seedthresh);
-  void buildSuperCluster(CalibratedClusterPtr&, CalibratedClusterPtrVector&);
+  void buildAllSuperClustersMustache(CalibratedClusterPtrVector&, double seedthresh);
+  void buildAllSuperClustersDeepSC(CalibratedClusterPtrVector&, double seedthresh);
+  void buildMustache(CalibratedClusterPtr&, CalibratedClusterPtrVector&);
```

Comment:
minor comment #2: this could be called `buildMustacheSuperCluster` or `buildSuperClusterMustache` ?
```suggestion
  void buildMustacheSuperCluster(CalibratedClusterPtr&, CalibratedClusterPtrVector&);
```